### PR TITLE
added R script to generate graphs

### PR DESCRIPTION
**Pull request by tirpidz**
* still need to handle the unity graphs

### CODE REVIEW

File: tools/graph_generation/grapher.r [hunk 1]
```diff
@@ -0,0 +1,63 @@
+install.packages("ggplot2")
+library("ggplot2")
+library("plyr")
+
+setwd("C:\\Users\\tirpi\\Downloads")
```
Comment (nfrechette) [inline tools/graph_generation/grapher.r]: wow very cryptic syntax haha, looks good though, only 2 small things: can we load this path from the command line?

File: tools/graph_generation/grapher.r [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ue4_stats <- read.csv("ue4_stats.csv", row.names = NULL)
+acl_stats <- read.csv("acl_stats.csv", row.names = NULL)
```
Comment (nfrechette) [inline tools/graph_generation/grapher.r]: or perhaps read those two paths from the command line (one, or the other, or both perhaps?)

File: tools/graph_generation/grapher.r [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+plot(p5)
+dev.off()
+
+p6 <- ggplot(bundle, aes(x = max_error, color = name)) + geom_histogram(fill = "white", alpha = 0.5, position = "identity") +
+geom_vline(data = mu_error, aes(xintercept = grp.mean, color = name), linetype = "dashed") +
+theme(legend.position = "top")
+
+png(filename = "ue4_acl_max_error.png")
+plot(p6)
+dev.off()
```
Comment (nfrechette) [inline tools/graph_generation/grapher.r]: newline at the end

File: tools/graph_generation/grapher.r [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+bundle <- data.frame(name, compression_ratio, max_error)
+mu_ratio <- ddply(bundle, "name", summarise, grp.mean = mean(compression_ratio))
+mu_error <- ddply(bundle, "name", summarise, grp.mean = mean(max_error))
+
+p1 <- ggplot(ue4_stats, aes(x = Compression.Ratio)) +
+  geom_histogram(color = "black", fill = "white") +
+  geom_vline(aes(xintercept = mean(Compression.Ratio)), color = "blue", linetype = "dashed", size = 1)
+
+png(filename = "ue4_compression_ratio.png")
```
Comment (nfrechette) [inline tools/graph_generation/grapher.r]: might be worth checking to make sure the png generation settings enable compression or use the proper bit rate, not sure what the default settings are, //todo

File: tools/graph_generation/grapher.r (hunk 3 above)
Comment (tirpidz) [inline tools/graph_generation/grapher.r]: fixed

File: tools/graph_generation/grapher.r (hunk 2 above)
Comment (tirpidz) [inline tools/graph_generation/grapher.r]: they are being read from the working dir which is the only required argument of the script

File: tools/graph_generation/grapher.r (hunk 4 above)
Comment (tirpidz) [inline tools/graph_generation/grapher.r]: https://stat.ethz.ch/R-manual/R-devel/library/grDevices/html/png.html

Comment (tirpidz) [inline tools/graph_generation/grapher.r]: y'a des optoins qui te semblent plus sexy que les autres?

**Pull request merged**